src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="index.html"
+      class="form"
+    >
+      <input
+        class="input"
+        type="text"
+        name="search"
+        id="search"
+        placeholder="Try&quot;Los Angeles&quot;"
+        data-qa="big"
```

Comment:
you should add this attribute to form, not to input

Reply:
please read requirements: `add attribute data-qa="keypress" to input in big search form
`